io.github.nyre221.kiview.json

```diff
@@ -0,0 +1,45 @@
+{
+   "base": "io.qt.qtwebengine.BaseApp",
+   "base-version": "6.6",
+   "command": "kiview",
+   "build-options": {
+    "env": [
+      "QTWEBENGINE_DISABLE_SANDBOX=1"
+    ]
+  },
+
+   "finish-args": [
+      "--share=ipc",
+      "--device=dri",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--env=QML_IMPORT_PATH=/app/qml" ,
+      "--socket=pulseaudio",
+      "--socket=session-bus",
```

Comment:
`session-bus` is not allowed nor did you justify.

Reply:
It is used to obtain the list of all Dolphin windows and the name of their interface.
Example in bash:
`$dbus-send --session --print-reply --type=method_call --dest=org.freedesktop.DBus /org/freedesktop/DBus org.freedesktop.DBus.ListNames | grep -i dolphin `
`$  string "org.kde.dolphin-13165"`

I couldn't find another way to access `ListNames`

My application needs it to interact with dolphin and send signals to the top window.

As I said before, I can remove it from the .json file and write it in the guide to set up the shortcut.

Reply:
Yes listnames is filtered inside the sandbox so that apps cannot know about applications running outside https://github.com/flatpak/xdg-dbus-proxy/blob/7466c8137fc06f863fde8486521984e43a26cd10/flatpak-proxy.c#L2032

Reply:
If this stays it needs an exception too https://docs.flathub.org/docs/for-app-authors/linter#finish-args-arbitrary-dbus-access

Reply:
> It is used to obtain the list of all Dolphin windows and the name of their interface.

Also why exactly do you want this information? What's the use-case here?

Reply:
Kiview needs to send commands to the active dolphin window to know which file the user wants to view (if there is a file selected), or to deduce which folder the shortcut was invoked in (so it can display a preview of the folder).

Example with the python version of kiview (QuickView): https://www.youtube.com/watch?v=mCfPZ5dgIZs